Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-interfaces.controller.ts

```diff
@@ -1,142 +1,42 @@
                             var interfaces = result.data.response;
-                            _.each(interfaces, function (data: { exhibitedById; usage; }) {
-                                $http.get("api/itInterfaceExhibitUsage/?usageId=" + itSystemUsage.id + "&exhibitId=" + data.exhibitedById).success(function (usageResult) {
+                            _.each(interfaces, (data: { exhibitedById; usage; }) => {
+                                $http.get(`api/itInterfaceExhibitUsage/?usageId=${itSystemUsage.id}&exhibitId=${data.exhibitedById}`).success(usageResult => {
                                     data.usage = usageResult.response;
                                 });
                             });
                             return interfaces;
-                        });
-                    }
+                        })
                 ],
                 user: [
-                    "userService", function (userService) {
-                        return userService.getUser();
-                    }
+                    "userService", userService => userService.getUser()
                 ]
             }
         });
     }]);
 
-    app.controller("system.EditInterfaces", ["$rootScope", "$scope", "$http", "notify", "tsas", "interfaces", "interfaceTypes", "methods", "dataTypes", "frequencies", "itSystemUsage", "exhibits",
-        function ($rootScope, $scope, $http, notify, tsas, interfaces, interfaceTypes, methods, dataTypes, frequencies, itSystemUsage, exhibits) {
+    app.controller("systemUsage.ExposedInterfaces", ["$scope", "itSystemUsage", "exhibits",
+        ($scope, itSystemUsage, exhibits) => {
 
-            $scope.frequencies = frequencies;
-
-            $scope.filterInterfaces = function (interfaceSystem) {
-                return interfaceSystem.canBeUsed || interfaceSystem.usage || $scope.showAllInterfaces;
-            };
-
-            // resolving complex types from ids
-            function resolveTypes(theInterface) {
-                theInterface.interfaceType = _.find(interfaceTypes, { Id: theInterface.interfaceTypeId });
-                theInterface.interface = _.find(interfaces, { Id: theInterface.interfaceId });
-                theInterface.method = _.find(methods, { Id: theInterface.methodId });
-                theInterface.tsa = _.find(tsas, { Id: theInterface.tsaId });
-
-                _.each(theInterface.dataRows, function (dataRow: { dataTypeId; dataType; }) {
-                    dataRow.dataType = _.find(dataTypes, { Id: dataRow.dataTypeId });
-                });
-
-                theInterface.numRows = theInterface.dataRows.length;
-            }
+            $scope.systemUsage = itSystemUsage;
 
-            // interface exposures
-            _.each(exhibits, function (interfaceExposure: { updateUrl; urlParams; id; }) {
+            // Enables marking "wished" 
```

Comment:
Er den kommentar stadig gyldog?

Reply:
Det er mig der har lavet den. For at man ved hvad det stykke kode gør